compute_transfer_examples/README.md

```diff
@@ -0,0 +1,165 @@
+
+**Example scenario:**
+- Your GCS collection has configured its base path to `/path/to/root/`.
+- A user wants to tar the files at the absolute path `/path/to/root/input_files/`.
+- To both the user and Flows service, this path appears as `/input_files/` on the GCS collection.
+- However, the Compute function running on the shared GCS instance **does not know** about the collection's configured base path and can only find the files using absolute paths.
+
+Thus, the Compute function must be provided with the GCS collection's configured base path to do the necessary transformations. In this example, `gcs_base_path` would need to be set to `/path/to/root/`.
+
+## Compute and Transfer Flow: Example 1
+In the first example, the Compute and Transfer flow takes a user-provided list of source files that **already** exists in the co-located GCS collection, creates a tarfile from them, and transfers the tarfile to a user-provided destination collection. Specifically, the flow will:
+1. Set constants for the run
+2. Create an output directory named after the flow's run ID on your GCS collection
+3. Invoke the Compute function `do_tar` on the source endpoint to create a tar archive from the input source files and save it in the output directory
+4. Transfer the resulting tarfile to the destination collection provided in the flow input
+5. Delete the output directory
+
+### Registering the Flow
+
+1. Edit `compute_transfer_examples/compute_transfer_example_1_definition.json` and replace the placeholder values:
+   - `gcs_endpoint_id`: Your GCS Collection ID
+   - `compute_endpoint_id`: Your Compute Endpoint ID
+   - `compute_function_id`: The UUID of the registered `do_tar` function
+
+If your GCS collection has a configured base path, also edit `gcs_base_path`.
+
+
+2. Register the flow:
+   ```bash
+   globus flows create "Compute and Transfer Flow Example 1" \
+     ./compute_transfer_examples/compute_transfer_example_1_definition.json \
+     --input-schema ./compute_transfer_examples/compute_transfer_example_1_schema.json
+   ```
+
+3. Save the Flow ID returned by this command
+
+### Running the Flow
+
+1. Create the flow input json file like so:
+   ```json
+   {
+       "source_paths": ["/path/to/file1", "/path/to/file2"],
+       "destination_path": "/path/to/your/destination/file.tar.gz",
+       "destination_endpoint_id": "your-destination-endpoint-uuid"
+   }
+   ```
+
+2. Start the flow:
+   ```bash
+   globus flows start <YOUR FLOW ID> \
+     --input <YOUR FLOW INPUT FILE> \
+     --label "Compute and Transfer Flow Example 1 Run"
+   ```
+   And save the run ID for use in the next command.
+
+3. Monitor the run progress:
+   ```bash
+   globus flows run show <RUN_ID>
+   ```
+   At this point, you might see that your flow has gone INACTIVE. This is because you need to give data access consents for any GCS collection that your flow is interacting with. Run the command:
```

Comment:
```suggestion
   At this point, you might see that your flow has become INACTIVE. This is because you need to give data access consents for any GCS collection that your flow is interacting with. Run the command:
```